### DISCUSSION TITLE
Live view in page "Update Revenue Arrangements and Revenue Plans"

### DISCUSSION THREAD

**Discussion by loicb24** [burkybang/SuiteAdvanced-Feedback#144] [Ideas]
Hi, 

I was wondering if it was possible to have the "live view" feature in the page "Update Revenue Arrangements and Revenue Plans" (Transaction -> Financial -> Update Revenue Arrangements and Revenue Plans): 

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/10929817/ad545e07-d982-40eb-84c4-54df2a9bde9a)

Thanks in advance, 

Loïc

**Comment by burkybang** [comment]
Probably. I'll check it out!

**Comment by burkybang** [comment]
This'll be in the next release. I've never used this page before, so I hope the Live View button works the way you envision. It does exactly what the existing Refresh button does but on an interval.

![image](https://github.com/burkybang/SuiteAdvanced-Feedback/assets/927886/36344fb2-1a1b-47d0-bfc7-ae6a8e7ecabe)

**Reply by burkybang** [reply]
@loicb24 The update is out now!

https://suiteadvanced.com/version-29_3_0
